### PR TITLE
Update performance.rst to use $container->getParameterBag()

### PR DESCRIPTION
**Pull request by gnumoksha**
there is no `$container->parameters()`

### CODE REVIEW

File: performance.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ container into a single file, which could improve performance when using
         // config/services.php
 
         // ...
-        $container->parameters()->set('.container.dumper.inline_factories', true);
+        $container->getParameterBag()->set('.container.dumper.inline_factories', true);
```
Comment (alamirault) [inline performance.rst]: I think here `$container` is a `ContainerConfigurator` and not `ContainerInterface`

If i'm right, we should probably update example like is done in the following sample

https://github.com/symfony/symfony-docs/blob/19ca84589284485ad490b617db7c4562ba5ac4d8/configuration.rst?plain=1#L375-L383

**Comment by OskarStark** [comment]
Friendly ping @gnumoksha

**Comment by gnumoksha** [comment]
I've changed the variable name, but I don't know how to verify if the statement was successful.

File: performance.rst [hunk 2]
```diff
@@ -81,7 +81,7 @@ container into a single file, which could improve performance when using
         // config/services.php
 
         // ...
-        $container->parameters()->set('.container.dumper.inline_factories', true);
+        $configurator->parameters()->set('.container.dumper.inline_factories', true);
```
Comment (xabbuh) [inline performance.rst:84]: I guess the whole example needs to look like this:

```rst
    .. code-block:: php

        // config/services.php
        namespace Symfony\Component\DependencyInjection\Loader\Configurator;

        return function(ContainerConfigurator $container): void {
            $container->parameters()->set('.container.dumper.inline_factories', true);
        };
```

Comment (javiereguiluz) [reply performance.rst:84]: I've done this change while merging. Thanks!

**Comment by javiereguiluz** [comment]
@gnumoksha thanks and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**